aldryn_forms/cms_plugins.py

```diff
@@ -395,6 +412,9 @@ class BooleanField(Field):
         'error_messages',
     ]
 
+    def serialize_value(self, instance, value):
+        return 'Yes' if value else 'No'
```

Comment:
I thought about that, but this would get serialized with the language of the user submitting the answer, is that ok?
(Currently we just serialize True/False)